<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWidgetUi</class>
 <widget class="QDialog" name="ConnectionWidgetUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect...</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QGroupBox" name="connGroupBox">
     <property name="title">
      <string>Connection settings</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="8" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="nameCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editConnection">
          <property name="toolTip">
           <string>edit this connection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../assets/default.qrc">
            <normaloff>:/assets/icons/icons8-edit-file-100.png</normaloff>:/assets/icons/icons8-edit-file-100.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="duplicateConnection">
          <property name="toolTip">
           <string>duplicate this connection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../assets/default.qrc">
            <normaloff>:/assets/icons/icons8-new-copy-100.png</normaloff>:/assets/icons/icons8-new-copy-100.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteConnection">
          <property name="toolTip">
           <string>delete</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../../../assets/default.qrc">
            <normaloff>:/assets/icons/icons8-delete-document-100.png</normaloff>:/assets/icons/icons8-delete-document-100.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testConnection">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="8" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <widget class="QLabel" name="textLabel5_2">
          <property name="text">
           <string>P&amp;ort</string>
          </property>
          <property name="buddy">
           <cstring>portSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="driverCombo"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>D&amp;river</string>
          </property>
          <property name="buddy">
           <cstring>driverCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="hostEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="textLabel3">
          <property name="text">
           <string>Database</string>
          </property>
          <property name="buddy">
           <cstring>databaseEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="portSpinBox">
          <property name="specialValueText">
           <string>3306</string>
          </property>
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="textLabel4">
          <property name="text">
           <string>&amp;Username</string>
          </property>
          <property name="buddy">
           <cstring>usernameEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="usernameEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="textLabel5">
          <property name="text">
           <string>Host</string>
          </property>
          <property name="buddy">
           <cstring>hostEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="passwordEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="textLabel4_2">
          <property name="text">
           <string>&amp;Password</string>
          </property>
          <property name="buddy">
           <cstring>passwordEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="databaseEdit"/>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="dbCheckBox">
       <property name="text">
        <string>Us&amp;e predefined in-memory database</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dbCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../assets/default.qrc"/>
 </resources>
 <connections/>
</ui>
